typesetting/aegisub.md

```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
 [Inkscape]: https://inkscape.org/en/
 [line0's builds]: https://files.line0.eu/builds/Aegisub/
 
-## Configuring Aegisub
 
-For now, just change your settings to reflect the following. If you've
-made any changes previously for another fansub role, be careful not to
-overwrite those. When in doubt, ask someone with Aegisub experience.
-Settings can be accessed via *View \> Options* or with the hotkey *Alt +
-O*.
-
-![aegisub64\_2017-09-08\_01-50-08.png](images/cnvimage100.png)
```

Comment:
Don't you want to delete these images? Their deletion isn't included in the commit (or GitHub is lying to me).

Reply:
Yes, didn’t know how to do it properly without having merging issues.

Reply:
There shouldn't be any merge issues if you aren't modfiying the same file/location in a text file in a different commit on a different branch.

Reply:
Oh, okay, I'll delete them in a new commit on this PR.